internal/cli/cli.go

```diff
@@ -194,9 +199,9 @@ func createStatusCommand(controller controller.ControllerInterface) *cobra.Comma
 	return &cobra.Command{
 		Use: "status",
 		Short: `List environments & services where the current repo is deployed. 
-This commands needs to be called from within a Git repo.`,
+                This commands needs to be called from within a Git repo.`,
```

Comment:
action - same comment. But not that I see that u actually fixed it, is it on purpose? U really want something like 

```
List environments & services where the current repo is deployed.
                      This commands needs to be called from within a Git repo.
```

? if yes and on purpose, then ignore all my action about that. Might wanna check with Coop and Blake too to make sure it's not too weird.